src/commands/hgetall.js

```diff
@@ -1,3 +1,3 @@
 export function hgetall(key) {
-  return this.data.get(key) || {};
+  return this.data.get(key) || [];
```

Comment:
good catch!

Reply:
👍  and this actually verifies that reply transformer is operational. hgetall reply transformer is converting array to object